app/src/main/java/org/systers/mentorship/view/activities/RequestDetailActivity.kt

```diff
@@ -134,14 +137,17 @@ class RequestDetailActivity: BaseActivity() {
         }
     }
 
-    private fun setObservables() {
+    private fun setObservables(relationResponse: Relationship) {
         requestDetailViewModel  = ViewModelProviders.of(this).get(RequestDetailViewModel::class.java)
         requestDetailViewModel.successful.observe(this, Observer {
             successful ->
             hideProgressDialog()
             if (successful != null) {
                 if (successful) {
                     Toast.makeText(this, requestDetailViewModel.message, Toast.LENGTH_LONG).show()
+                    val previousScreen = Intent(applicationContext, RequestPagerFragment::class.java)
+                    previousScreen.putExtra("requestid", relationResponse.id.toString())
```

Comment:
Just a little suggestion here, can you put that key "requestid" in Constants.kt class because all the constants are defined inside that class and it would be better to also define this constant in that class.
And change here to Constants.REQUEST_ID